docs/authentication/wallets/3id-connect.md

```diff
@@ -1,52 +1,139 @@
 # 3ID Connect
 
-3ID Connect is an authentication SDK for web applications that allows a user to control a [3ID DID](../dids/3id.md) using any number of blockchain accounts/wallets.
+3ID Connect is the most widely used [authentication]() provider for Ceramic. The 3ID Connect authentication SDK allows users to use a [3ID DID]() with their existing blockchain wallet(s). 3ID Connect was designed for developers building Ceramic-enabled web apps whose users already have a blockchain/crypto wallet. By using 3ID Connect, your users get to rely on the signing and key management capabilities of the crypto wallet(s) they already have instead of needing to install a separate wallet.
 
-3ID Connect allows users to:
-- Create a 3ID
-- Control their 3ID with one or more blockchain wallets
-- 
+**Want to install 3ID Connect?** Visit the [**usage**](#usage) section below.
+
+**Want to try 3ID Connect?** Visit [Self.ID](https://self.id) to try it in a live application.
+
+> 3ID Connect is not a substitute for a crypto wallet. It works alongside these wallets.
+
+#### Supported blockchains
+
+- Ethereum
+- EVM compatible L1s and L2s (i.e. Avalanche)
+- NEAR
+- Moonbeam
+- EOS
```

Comment:
I'm not sure we support all these, when I tried setting up integration tests I could only make it work with Ethereum.

Reply:
I think we do work with EVM chains, since they can just use metamask. Moonbeam is also an EVM chain. I wasn't aware that 3ID Connect wasn't working with NEAR or EOS tho @PaulLeCam

Reply:
Okay made this fix. Look good otherwise?